src/screens/WalletConnectApprovalSheet.js

```diff
@@ -1,30 +1,41 @@
 import { useRoute } from '@react-navigation/native';
 import analytics from '@segment/analytics-react-native';
+import { get } from 'lodash';
 import React, {
   useCallback,
   useEffect,
   useMemo,
   useRef,
   useState,
 } from 'react';
-import { InteractionManager } from 'react-native';
+import { InteractionManager, View } from 'react-native';
+import { ContextMenuButton } from 'react-native-ios-context-menu';
 import styled from 'styled-components';
+import ChainLogo from '../components/ChainLogo';
 import Divider from '../components/Divider';
 import { Alert } from '../components/alerts';
+import ButtonPressAnimation from '../components/animations/ButtonPressAnimation';
 import { RequestVendorLogoIcon } from '../components/coin-icon';
-import { Centered, Row } from '../components/layout';
+import { ContactAvatar } from '../components/contacts';
+import ImageAvatar from '../components/contacts/ImageAvatar';
+import { Centered, Column, Row } from '../components/layout';
 import {
   Sheet,
   SheetActionButton,
   SheetActionButtonRow,
 } from '../components/sheet';
 import { Text } from '../components/text';
+import { useTheme } from '@rainbow-me/context';
```

Comment:
`useTheme`  is already in the global scope so we don't need to import it on every file 😄